scm-ui/ui-webapp/public/locales/de/commons.json

```
@@ -116,6 +116,15 @@
         "displayName": "Systemstandard",
         "description": "Verwendet die Standardeinstellung des Systems"
       }
+    },
+    "accessibility": {
+      "navLink": "Barrierefreiheit",
+      "subtitle": "Barrierefreiheit",
+      "deactivateShortcuts": {
+        "label": "Tastaturkürzel deaktivieren",
+        "helpText": "Deaktiviert alle speziellen Tastaturkombinationen, welche die Barrierefreiheit beeinträchtigen könnten."
```

```suggestion
        "helpText": "Deaktiviert alle SCM-Manager Tastaturkombinationen, welche die Barrierefreiheit beeinträchtigen könnten."
```

Perhaps this is clearer?